src/test/kotlin/nl/info/client/zgw/model/ZrcFixtures.kt

```diff
@@ -73,6 +73,14 @@ fun createOpschorting(
     this.indicatie = indicatie
 }
 
+fun createOpschorting(
```

Comment:
There is already a `createOpschorting` function. Why create another one? Just extend /change the one that's already there.